source/numerics.tex

```diff
@@ -6946,23 +6953,22 @@
 
 \begin{itemdescr}
 \pnum
-\effects As if by: \tcode{*this = valarray(il);}
-
-\pnum
-\returns \tcode{*this}.
+\effects Equivalent to: \tcode{return *this = valarray(il);}
```

Comment:
I don't see any negative effect from using "Equivalent to" here either, although it's again arguably not editorial.